app/models/event.rb

```diff
@@ -0,0 +1,24 @@
+class Event < ApplicationRecord
+  belongs_to :move
+
+  enum event_types: {
+    move_created: 'move_created',
+    move_updated: 'move_updated',
+    move_completed: 'move_completed',
+    move_cancelled: 'move_cancelled',
+    move_redirected: 'move_redirected',
+    move_lockout: 'move_lockout',
+    journey_created: 'journey_created',
+    journey_updated: 'journey_updated',
+    journey_completed: 'journey_completed',
+    journey_uncompleted: 'journey_uncompleted',
+    journey_cancelled: 'journey_cancelled',
+    journey_uncancelled: 'journey_uncancelled',
+  }
+
```

Comment:
I've noticed this just now, it looks like we got 2 only `event_types`: `move`, `juorney`
and we also have an `action`: `create, update, cancel, restore, ...`

What do you think? 
it might be worth to refactor this, if not what about renaming event_types to event_names?

Reply:
There will be lots more events in future: GPS tracking locations; lodging, pickup and handovers. I'll think about breaking the name into two: `event_type` + `action`. 🤔

Reply:
👍